js/main.js

```diff
@@ -0,0 +1,24 @@
+// Доработано на основе: https://dev-gang.ru/article/javascript-generirovat-sluczainoe-czislo-v-diapazone-s1azn94f4k/
+const GET_RANDOM_NUMBER = (min, max) => {
```

Comment:
Константа это когда мы знаем значение до начала исполнения кода и во время исполнения оно не меняется, здесь у тебя обычная переменная в которую мы складываем объявление функции
```suggestion
const getRandomNumber = (min, max) => {
```